module.js

```diff
@@ -1,4 +1,5 @@
 const _ = require('lodash');
+const filter = require('./filters');
```

Comment:
can we rename file as filterBehavior or something similar? Also rename it while exporting. 
In future, we might have different behavior files for different parts of code as well.